docs/simulation_development.md

```diff
@@ -0,0 +1,28 @@
+# Adding new simulations
```

Comment:
It might be good to clarify what these simulations are. Perhaps note that these are used for performance testing. Or, re-title it to "Adding New Performance Simulations"